src/client/app/app.config.js

```diff
@@ -57,6 +57,20 @@
                 }
             },
             {
+                state: ' /offers/edit/{id}',
+                config: {
+                    url: '/editOffer',
```

Comment:
url should be here, state can be defined like `offers.edit`
this field should have `/offers/edit/{id}